pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -65,6 +76,8 @@ type manager struct {
 	podMap map[string]string
 	//Topology Manager Policy
 	policy Policy
+	//Topology Manager Scope
+	topologyScope string
```

Comment:
why dropping `manager` from the name of a field?

Reply:
we don't have `topologyManagerPolicy` too, just `policy` alone. I was considering leaving `scope` alone but `topologyScope` may be more explanatory, what do you think?

Reply:
right, it is part of a manager struct already. nvm

Reply:
Can we introduce a type like we have for `Policy` instead of just a raw string.

Reply:
IMO: It would complicate code a bit (type conversion probably). 
`func NewManager(numaNodeInfo cputopology.NUMANodeInfo, topologyPolicyName string, topologyScopeName string)` 
has a `topologyScopeName string` as a parameter so having `scope` (earlier `topologyScope`) as a string  seems to be a straightforward solution.